src/bin/cargo/commands/add.rs

```diff
@@ -76,6 +76,18 @@ Example uses:
                 "ignore-rust-version",
                 "Ignore `rust-version` specification in packages (unstable)"
             ),
+            flag("public", "Mark the dependency as public")
+                .long_help("Mark the dependency as public
+
+The dependnecy will be visible in both of your crate and outside."),
+            flag("no-public", "Mark the dependency as private")
+                .conflicts_with("dev")
```

Comment:
no-public has this twice

We need it on public and no-public as well as a conflict for `build`

Reply:
This is still missing from `public`

Reply:
Sorry by left out it twice.